src/labels/CustomLabels.labels

```diff
@@ -7264,6 +7264,78 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Title of checklist item Train Users in Get Started</shortDescription>
         <value>Train Your users</value>
     </labels>
+    <labels>
+        <fullName>gsChecklistItemWhatIsNPCloudLink</fullName>
```

Comment:
@dcaballeroSf If I remember correctly, didn't you want to use the `gseu` prefix on all the End User Get Started content?

Reply:
Yeap, I think we need to use that prefix to keep Get Started page separated